airsonic-main/src/main/webapp/WEB-INF/jsp/playlist.jsp

```diff
@@ -285,6 +286,16 @@
                     playlistMusicTable.one( "draw", function () {
                         onRearrange(playlistMusicTable.rows().indexes().toArray());
                     });
+                    playlistMusicTable.order([0, "asc"]);
+                }
+            });
+            playlistMusicTable.on("order", (e, settings, ordArr) => {
+                if (previousSortingOrder.col !== ordArr[0].col || previousSortingOrder.dir !== ordArr[0].dir) {
+                    playlistMusicTable.order([ordArr[0].col, ordArr[0].dir]);
```

Comment:
worried about a loop here.

would calling the 'order()' function within the 'order' event cause another 'order' event to be generated thus causing the loop?

Reply:
that's exactly the reason